arcgis-ios-sdk-samples/Augmented reality/Navigate in AR/README.md

```diff
@@ -52,13 +52,17 @@ When you start, route instructions will be displayed and spoken. As you proceed
 
 This sample uses Esri's [World Elevation](https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer) service to ensure that route lines are placed appropriately in 3D space. It uses Esri's [World Route](https://www.arcgis.com/home/item.html?id=1feb41652c5c4bd2ba5c60df2b4ea2c4) service to calculate routes. The world routing service requires authentication and does consume ArcGIS Online credits.
 
-**Note:** This item requires an ArcGIS Online organizational subscription or an ArcGIS Developer account and consumes credits.
+**Note:** This routing service requires one of the following authentication methods
 
-To access this sample, you'll need to do one of the following:
+* An ArcGIS Online organizational subscription and consumes credits
```

Comment:
```suggestion
* An ArcGIS Online organizational subscription and credits
```
Hmm, seems like "consumes credits" is being used as a noun here but to my knowledge, there isn't a proper term for Esri credits. I'd suggest just leaving it like this.